pydc_control/cli.py

```diff
@@ -278,7 +278,7 @@ def _validate_args(args: argparse.Namespace) -> int:
             f'To use the config service, you must not be developing on the '
             f'{config_service_target.project_name} project.'
         )
-        return os.EX_USAGE
+        # return os.EX_USAGE
```

Comment:
This comment can probably be removed

Reply:
Good catch, whoops! This is what I originally started testing out/improving...